apps/protocol-frontend/src/components/DaoSettingsMembersTable.tsx

```diff
@@ -97,21 +108,32 @@ const DaoSettingsMembersTable = ({ daoId }: DaoSettingsMembersTableProps) => {
         ),
       },
       {
-        header: 'Member Address',
+        header: 'Member',
         accessorKey: 'user',
         cell: ({ getValue }: { getValue: Getter<UIGuildUsers['user']> }) => {
           const value = getValue();
+          const displayMemberName =
+            value.name || value.display_name || displayAddress(value.address);
           return value.address === userData?.address ? (
             <Text
               whiteSpace="normal"
               fontWeight="bold"
               bgGradient="linear(to-l, #7928CA, #FF0080)"
               bgClip="text"
             >
-              {value.address}
+              {displayMemberName}
             </Text>
+          ) : value.name || value.display_name ? (
+            <Tooltip
+              variant="primary"
+              label={displayAddress(value.address)}
```

Comment:
Shouldn't this be the full address

Reply:
Isn't the goal here to verify the address

Reply:
That's what I thought too -- I think I asked this yesterday but maybe I wasn't clear. I agree that this should be full and if it's displayed in the table it's truncated.

Reply:
I removed the truncate here since I agree -- if it's too cumbersome we can revisit something like positioning the tooltip